app/dao/notifications_dao.py

```diff
@@ -136,6 +135,28 @@ def dao_get_7_day_agg_notification_statistics_for_service(service_id,
 
 
 @statsd(namespace="dao")
+def dao_get_template_usage(service_id, limit_days=None):
```

Comment:
actually it's a bit gnarly but could do that keeping limit_days, but just saying

```
notification_table = Notification if limit_days == '7' else NotificationHistory
...
db.session.query(
    notification_table.template_id
    ...
```